### PR TITLE
Fixing tests and other small errors

### PR DESCRIPTION
**Pull request by lassoan**
This is a collection of unrelated small fixes bundled together in one pull request for joint review.

### CODE REVIEW

File: Libs/MRML/Core/vtkMRMLCrosshairNode.cxx [hunk 1]
```diff
@@ -52,161 +40,28 @@ void vtkMRMLCrosshairNode::WriteXML(ostream& of, int nIndent)
...
-    }
-
-  of <<  " crosshairRAS=\"" << this->CrosshairRAS[0] << " "
-     << this->CrosshairRAS[1] << " " << this->CrosshairRAS[2] << "\"";
+  vtkMRMLWriteXMLBeginMacro(of);
+  vtkMRMLWriteXMLEnumMacro(crosshairMode, CrosshairMode);
+  vtkMRMLWriteXMLEnumMacro(crosshairBehavior, CrosshairBehavior);
+  vtkMRMLWriteXMLEnumMacro(crosshairThickness, CrosshairThickness);
+  vtkMRMLWriteXMLVectorMacro(crosshairRAS, CrosshairRAS, double, 3);
+  vtkMRMLWriteXMLBooleanMacro(fastPick3D, FastPick3D);
```
Comment (jcfr) [inline Libs/MRML/Core/vtkMRMLCrosshairNode.cxx:48]:
```suggestion
  // This property is only for evaluation of this feature and the value is not stored persistently in the scene file.
  // vtkMRMLWriteXMLBooleanMacro(fastPick3D, FastPick3D);
```

File: Libs/MRML/Core/vtkMRMLCrosshairNode.cxx [hunk 2]
```diff
@@ -52,161 +40,28 @@ void vtkMRMLCrosshairNode::WriteXML(ostream& of, int nIndent)
...
-    }
-
-  this->EndModify(disabledModify);
-
+  vtkMRMLReadXMLBeginMacro(atts);
+  vtkMRMLReadXMLEnumMacro(crosshairMode, CrosshairMode);
+  vtkMRMLReadXMLEnumMacro(crosshairBehavior, CrosshairBehavior);
+  vtkMRMLReadXMLEnumMacro(crosshairThickness, CrosshairThickness);
+  vtkMRMLReadXMLVectorMacro(crosshairRAS, CrosshairRAS, double, 3);
+  vtkMRMLReadXMLBooleanMacro(fastPick3D, FastPick3D);
```
Comment (jcfr) [inline Libs/MRML/Core/vtkMRMLCrosshairNode.cxx:63]:
```suggestion
  // This property is only for evaluation of this feature and the value is not stored persistently in the scene file.
  // vtkMRMLReadXMLBooleanMacro(fastPick3D, FastPick3D);
```

File: Libs/MRML/DisplayableManager/vtkMRMLCameraWidget.cxx [hunk 3]
```diff
@@ -516,21 +516,24 @@ bool vtkMRMLCameraWidget::ProcessSetCrosshair(vtkMRMLInteractionEventData* event
...
     }
-  double worldPos[3] = { 0.0 };
-  eventData->GetWorldPosition(worldPos);
 
-  vtkMRMLScene* scene = this->GetCameraNode()->GetScene();
-  vtkMRMLCrosshairNode* crosshairNode = vtkMRMLCrosshairDisplayableManager::FindCrosshairNode(scene);
-  if (!crosshairNode)
+  if (!crosshairNode->GetFastPick3D())
+    {
+    // Get accurate world position (by default, world position is retrieved from Z buffer of the renderer, which may be inaccurate)
```
Comment (jcfr) [inline Libs/MRML/DisplayableManager/vtkMRMLCameraWidget.cxx:528]:
```suggestion
    // Get accurate world position (by default FastPick3D is false and the world position is retrieved from Z buffer of the renderer, which may be inaccurate)
```

File: Modules/CLI/ExecutionModelTour/ExecutionModelTour.cxx [hunk 4]
```diff
@@ -14,7 +14,7 @@
 #include <vtkMRMLMarkupsFiducialNode.h>
 #include <vtkMRMLMarkupsFiducialStorageNode.h>
```
Comment (jcfr) [inline Modules/CLI/ExecutionModelTour/ExecutionModelTour.cxx:15]: `BUG: Fixed ExecutionModelTourTest` -> `BUG: Fix ExecutionModelTourTest`

File: SuperBuild.cmake [hunk 5]
```diff
@@ -339,7 +339,7 @@ list_conditional_append(Slicer_BUILD_DataStore Slicer_REMOTE_DEPENDENCIES DataSt
```
Comment (jcfr) [inline SuperBuild.cmake:339]: Update commit message from

```
BUG: Update CompareVolumes and LandmarkRegistration to fix VTK9 incompatibilities
```

to 

```
BUG: Update CompareVolumes and LandmarkRegistration to fix VTK9 incompatibilities

VTK9 does not provide vtk.VTK_MAJOR_VERSION.
```

File: Libs/MRML/Core/vtkMRMLCrosshairNode.cxx (hunk 1 above)
Comment (lassoan) [reply Libs/MRML/Core/vtkMRMLCrosshairNode.cxx:48]: Makes sense, I'll change this.

File: Libs/MRML/Core/vtkMRMLCrosshairNode.cxx (hunk 2 above)
Comment (lassoan) [reply Libs/MRML/Core/vtkMRMLCrosshairNode.cxx:63]: OK

File: Libs/MRML/DisplayableManager/vtkMRMLCameraWidget.cxx (hunk 3 above)
Comment (lassoan) [reply Libs/MRML/DisplayableManager/vtkMRMLCameraWidget.cxx:528]: OK

File: Modules/CLI/ExecutionModelTour/ExecutionModelTour.cxx (hunk 4 above)
Comment (lassoan) [reply Modules/CLI/ExecutionModelTour/ExecutionModelTour.cxx:15]: OK

File: SuperBuild.cmake (hunk 5 above)
Comment (lassoan) [reply SuperBuild.cmake:339]: OK

File: Libs/MRML/Core/vtkMRMLCrosshairNode.cxx (hunk 1 above)
Comment (jcfr) [reply Libs/MRML/Core/vtkMRMLCrosshairNode.cxx:48]: Looks like the `fastPick3D` property is still serialized ?

Comment (lassoan) [reply Libs/MRML/Core/vtkMRMLCrosshairNode.cxx:48]: Sorry, it seems that some of the changes have been discarded during rebase or force-push.

**Comment by lassoan** [comment]
Force-pushed again, this should be good now.

**Review by jcfr — APPROVED** [review]

File: Modules/CLI/ExecutionModelTour/ExecutionModelTour.cxx [hunk 6]
```diff
@@ -142,12 +142,15 @@ int main(int argc, char *argv[])
...
-  // coordinate system
-  outputFiducialStorageNode->UseLPSOn();
-  outputFiducialStorageNode->WriteData(copiedFiducialNode.GetPointer());
+  if (!seedsOutFile.empty())
+    {
+    vtkNew<vtkMRMLMarkupsFiducialStorageNode> outputFiducialStorageNode;
+    outputFiducialStorageNode->SetFileName(seedsOutFile.c_str());
+    // the .xml file specifies that it expects the output file in LPS
+    // coordinate system
+    outputFiducialStorageNode->UseLPSOn();
```
Comment (jcfr) [inline Modules/CLI/ExecutionModelTour/ExecutionModelTour.cxx:151]: This is independent of this PR but I am wondering if there is way to get the value associated with the `coordinateSystem` attribute:

https://github.com/Slicer/Slicer/blob/8db9e8e573b73d82384db97633697bda5aee4401/Modules/CLI/ExecutionModelTour/ExecutionModelTour.xml#L206-L212

**Pull request merged**